### ISSUE TITLE
CLI error can be confusing

### ISSUE THREAD

**Issue by arokem** [uwescience/myria-ec2-ansible#24]
For example, when I run: 

```
(py2) D-140-142-30-42:~  $myria-cluster create arokem
```

I get the following error:

```
No default VPC is configured for your AWS account in the 'us-west-2' region.
Please ask your administrator to create a default VPC or specify a VPC using the `--vpc-id` option.
```

But when running: 

```
(py2) D-140-142-30-42:~  $myria-cluster create arokem --vpc-id vpc-9a3c46cc
```

I get: 

```
Error: no such option: --vpc-id
```

OTOH, running: 

```
myria-cluster create arokem --subnet-id subnet-c655998
```

Does start up the machines on EC2. 

Could this be a case of the error handling lagging after changes in the code?

**Comment by bernease** [comment]
This is a problem with the eScience account -- we have an EC2 Classic
account (vs EC2 VPC). It means that there is no default VPC in most
regions. Thus, you need to specify a subnet id like you did in your example.

I've tried to wrangle the various parties needed to transfer the account in
the past, but we weren't able to get a time that worked and have AWS
onboard.

If this is very necessary, I'd love to pull together examples so we can
escalate with the people who are reliant on the eScience AWS account
(mainly viziometrics website and one other).

**Comment by senderista** [comment]
fixed in c3b35b9
